CSharp/DevVmPowershell/Helpers/ConnectionHelper.cs

```diff
@@ -36,5 +36,20 @@ public ServiceFactory GetServiceFactory(string protocol = Constants.Connection.P
 
 			return _serviceFactory;
 		}
+
+		public string GetUserName()

```

Comment:
Instance of 3 methods, can we just have one method which returns and instance of ImportApi class? That way it will be cleaner i think.